app/tournament/match.js

```diff
@@ -0,0 +1,39 @@
+export class Match {
+    constructor(round) {
+        this.round = round;
+        this.players = [];
+        this.isFinished = false;
+    }
+
+    addPlayer(player) {
+        if (this.players.length >= 2) {
+            throw new Error("Match has already 2 players");
+        }
+        this.players.push(player);
+    }
+
+    setNextMatch(match) {
+        this.nextMatch = match;
+    }
+
+    setWinner(player) {
```

Comment:
Is player expected to be a number? Use index or playerIndex instead.
Or even better, let player be a player.